index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-container">
+        <div class="logo">
+          <a href="#" id="welcome-logo">Welcome</a>
+        </div>
+        <nav class="mobile-menu-btn">
+          <img src="./icons/mobile-menu-btn.svg" alt="mobile menu" />
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="headline">
+        <div class="primary-headline">
+          <h1>
+            Hey there. I'm Darik
+            <span class="primary-color">I'm a software developer</span>
```

Comment:
- [ ] Kindly put the subtitle uses a header tag hx (examples: <h1>,<h2>,<h3>, etc. not in a `span` inside an h1 tag